pkg/pkg_test.go

```diff
@@ -22,9 +26,9 @@ var _ = Describe("Pkg", func() {
 			machineTypes, err = convert.CsvToResources("test/test.csv")
 			Expect(err).To(BeNil())
```

Comment:
This code is not part of this PR, but it should be
```suggestion
			Expect(err).NotTo(HaveOccurred())
```